code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -747,7 +747,7 @@
 	var/turf/T = get_turf(target)
 	if(T)
 		for(var/obj/effect/decal/cleanable/blood/B in view(T, 2))
-			if(B.blood_state == "blood")
+			if(B.blood_state == /datum/reagent/blood)
```

Comment:
Is this correct ? Looks like some icon state definition.

Reply:
the original code is also wrong, it should be `BLOOD_STATE_HUMAN`